pkg/sql/sqlbase/system.go

```diff
@@ -31,8 +31,9 @@ import (
 // `TestSystemTableLiterals` test that compares the table generated by
 // evaluating the `CREATE TABLE` statement to the descriptor literal that is
 // actually used at runtime.
+
+// These system tables are part of the system config.
 const (
-	// NamespaceTableSchema is checked in TestSystemTables.
 	NamespaceTableSchema = `
```

Comment:
given that these are exported const I'm surprised make check succeeds without a comment describe them.